### PR TITLE
Support fail-safe options; minor improvements

### PR DESCRIPTION
**Pull request by adrianwong**

### CODE REVIEW

File: PrinceXML.Wrapper/Prince.cs [hunk 1]
```diff
@@ -505,7 +505,8 @@ private List<string> GetJobCommandLine(string logType)
 
             foreach ((string k, string v) in Options)
             {
-                cmdLine.Add(ToCommand(k, v));
+                string val = v == "" ? null : v;
```
Comment (mikeday) [inline PrinceXML.Wrapper/Prince.cs:508]: Is this going to cause trouble for any options where we _want_ to allow empty string? `--user-password` perhaps?

**Comment by mikeday** [comment]
One issue is whether we need to support the more fine-grained options in the future, like `--fail-dropped-content`, such that FailSafe would just be a shorthand for setting all of them.

File: PrinceXML.Wrapper/Prince.cs (hunk 1 above)
Comment (adrianwong) [reply PrinceXML.Wrapper/Prince.cs:508]: Right, hadn't considered that. Looks like it's back to the original idea of updating the documentation then.

**Comment by adrianwong** [comment]
> One issue is whether we need to support the more fine-grained options in the future, like `--fail-dropped-content`, such that FailSafe would just be a shorthand for setting all of them.

Sure, might as well do that now I suppose.

**Review by mikeday — APPROVED** [review]
That looks good!

**Pull request merged**